app/scripts/modules/core/src/cluster/filter/clusterFilter.service.ts

```diff
@@ -118,7 +118,7 @@ export class ClusterFilterService {
     this.clusterFilterModel.asFilterModel.addTags();
```

Comment:
the line above this one `this.sortGroupsByHeading` sorts the `asFilterModel.groups` array in place using the local variable `groups` as a reference